src/data-catalog/DataBrowserDetails.ts

```diff
@@ -46,15 +48,22 @@ const styles = {
   cloudIconProps: { role: 'img', style: { maxHeight: 25, maxWidth: 150 } },
 };
 
-const MetadataDetailsComponent = ({ dataObj, name }) => {
+interface MetadataDetailsComponentProps {
+  dataObj: Dataset;
+}
+
+const MetadataDetailsComponent = ({ dataObj }: MetadataDetailsComponentProps) => {
   return h(Fragment, [
-    h2({ className: 'sr-only' }, [` ${name} Metadata`]),
+    h2({ className: 'sr-only' }, [` ${dataObj['dct:title']} Metadata`]),
```

Comment:
Was `name` not correctly provided before? I don't see where it was done in the caller.

Reply:
It wasn't! I think we missed it because its a sr-only class